### DISCUSSION TITLE
Multiple option picker do not resize when side-by-side

### DISCUSSION THREAD

**Discussion by gerhardmpl** [Budibase/budibase#5363] [Help] [answered]
To optimize screen usage, I would like to have three drop-down picker next to each other (direction row) and not on top of each other (direction column). 

- Scenario A - Using the option picker component with column type `Option`, three option field are evenly spread (when setting _width_ to 30%).
- Scenario B - Using the option picker component with a data provider / picker combination, the option fields _width_ is determined by the fields `label` and cannot be resized.

The example shows the different behaviour. How can I make the data provider / picker combination resize like scenario A for a better readability while beeing able to customize the options?

[example.txt](https://github.com/Budibase/budibase/files/8471918/example.txt)

**Comment by gerhardmpl** [comment]
Ok, an answer to #5354 gave me a hint to try absolute values for the pickers _width_ (e.g. 180px) instead of a percentages. While this will not be as responsive (e.g. on small displays), it is still a workaround.

**Comment by melohagan** [comment, marked as answer]
Hey @gerhardmpl 

You can click *Edit custom CSS* for the picker and enter `width: 30%` for the data providers.

**Reply by gerhardmpl** [reply]
That is a better workaround, thank you! Is that the preferred (albeit a little hidden) solution or would I rather be using the pickers width?

**Reply by melohagan** [reply]
Editing the pickers width is definitely more intuitive, but as in this case it may not always work depending on the parent containers.
